cfgov/paying_for_college/views.py

```diff
@@ -309,6 +311,24 @@ def school_search_api(request):
     return HttpResponse(json_doc, content_type='application/json')
 
 
+def school_autocomplete(request):
+    document = []
+    search_term = request.GET.get('q', '').strip()
+    if search_term:
+        response = SchoolSearch(search_term).autocomplete()
+
+        document = [{'schoolname': school.text,
+                     'id': school.school_id,
+                     'city': school.city,
+                     'nicknames': str(school.nicknames),
```

Comment:
No longer need to force the list to a string:

```suggestion
                     'nicknames': school.nicknames,
```

Reply:
Removed str from nicknames in school_autocomplete because list-as-string is not friendly in the API response